pkg/cache/multi_namespace_cache.go

```diff
@@ -305,42 +305,63 @@ type multiNamespaceInformer struct {
 	namespaceToInformer map[string]Informer
 }
 
+type handlerRegistration struct {
+	handles map[string]toolscache.ResourceEventHandlerRegistration
+}
+
+type syncer interface {
+	HasSynced() bool
+}
+
+// HasSynced asserts that the handler has been called for the full initial state of the informer.
+// This uses syncer to be compatible between client-go 1.27+ and older versions when the interface changed.
+func (h handlerRegistration) HasSynced() bool {
```

Comment:
I think I am blind, where do we call this?

Reply:
We don't call this, if allows the interface to be implemented on 0.27

Reply:
Since they added this function